core/src/main/web/app/controlpanel/controlpanel-directive.js

```diff
@@ -60,6 +63,17 @@
           return bkMenuPluginManager.getMenus();
         };
 
+        if (bkUtils.isElectron){
+          window.addEventListener('focus', function() {
```

Comment:
Well the event listener should stay in both control panel and mainapp. Does registering different handlers for the same event overwrite previous handlers? Or do they build up and leak?